libs/common-utilities/src/constants/proposals.ts

```diff
@@ -0,0 +1,12 @@
+export const PROPOSAL_STATUSES = {
+  unsponsored: 'Unsponsored',
+  voting: 'Voting',
+  grace: 'Grace',
+  expired: 'Expired',
+  cancelled: 'Cancelled',
+  needsProcessing: 'Ready for Processing',
+  failed: 'Failed',
+  passed: 'Passed',
+  actionFailed: 'Processing Failed',
+  unknown: 'Unknown',
+} as const;
```

Comment:
Is this being used anywhere? Or is a preemptive hub utility?

Reply:
> 

oh yeah. got ahead of myself there.